.github/workflows/test.yml

```diff
@@ -9,7 +9,8 @@ jobs:
   linux-tests:
     strategy:
       matrix:
-        node: [12, 14, 16, 18]
+        # TODO: remove older versions of node
+        node: [12, 14, 16, 18, 20, 22]
```

Comment:
Just curious - Is there a specific reason why we want to remove older node versions ins a separate PR?

Reply:
Yes, the plan is to remove the older node versions when we drop official support for them, which is a breaking change, so we plan to remove them in the next major version most likely when we release the OAS SDK